acceptance/options/embedded_db.rb

```diff
@@ -3,6 +3,18 @@
 #  a variable 'options_file_path', though.
 require File.expand_path(File.join(File.dirname(options_file_path), 'common.rb'))
 
-common_options_hash.merge({
-  :puppetdb_database => 'embedded',
-})
+defaults_opts = if options[:type] == 'aio' then
```

Comment:
@kbarber the eval in the options parser did not seem to like typical conditional statements. I wound up using the tap method to influence the merge on an item-by-item basis. It looks a little weird, but it works. Let me know if you have a better recommendation.

FYI: We will need to update the jenkins job(s) to use the correct options when the `puppetserver_embedded_db.rb` is deleted